data/Language/JavaScript.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: JavaScript
+wikidata: Q2005
+aliases:
+  - js
+  - ecmascript
+  - node
```

Comment:
`node` is definitely not an alias for `JavaScript`. `ecmascript` isn't either IMO, because 'JavaScript is a general-purpose scripting language that conforms to the ECMAScript specification.'

Reply:
@li-boxuan I followed the definition here: https://github.com/coala/coala/blob/master/coalib/bearlib/languages/definitions/JavaScript.py

Reply:
Hmmm, nvm then. If that is indeed a problem we can fix it later.

Reply:
IMHO that `ecmascript` shouldn't be an alias of JavaScript. It is a scripting standard. I also don't think that `node` is an alias of JavaScript.

As for now, just ignore this matter and move on. I think we could creates another issue for this matter.